src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The classes `calendar--start-day-sun` and `calendar--month-length-31` seem to be modifiers in BEM methodology. However, it's not clear what they modify. If they're used to set the start day of the week and the number of days in a month, it would be better to handle this logic with JavaScript. It would make your code more flexible and easier to maintain.